test/API/CMakeLists.txt

```diff
@@ -161,109 +169,6 @@ if (HDF5_ENABLE_FORMATTERS)
   clang_format (HDF5_TEST_h5_api_test_FORMAT h5_api_test)
 endif ()
 
-if (HDF5_TEST_API_INSTALL)
```

Comment:
Moved the install section to the bottom of this file for consistency with elsewhere